src/session.rs

```diff
@@ -130,9 +130,11 @@ impl Session {
         let chain = registry.get_chain(&self.config.chain_id).unwrap();
         let (_, request_state) = parse_request(&request)?;
         let mut chain_state = chain.state.lock().unwrap();
+        println!("aaaaa");
```

Comment:
?

Reply:
It's just for debugging, I already removed it.